src/lib/vav.ts

```diff
@@ -0,0 +1,102 @@
+import {Zcl} from "zigbee-herdsman";
+import * as exposes from "./exposes";
+import {logger} from "./logger";
+import {deviceAddCustomCluster} from "./modernExtend";
+import * as reporting from "./reporting";
+import type {KeyValueAny, ModernExtend} from "./types";
+import * as utils from "./utils";
+
+const e = exposes.presets;
+const NS = "zhc:nodon";
+
+const PILOT_WIRE_CLUSTER = "customClusterVavPilotWire";
+const manufacturerOptions = {manufacturerCode: 0x153a};
+
+interface VavPilotWire {
+    attributes: {
+        mode: number;
+    };
+    commands: {
+        setMode: {mode: number};
+    };
+    commandResponses: never;
+}
+
+const pilotWireCluster = deviceAddCustomCluster(PILOT_WIRE_CLUSTER, {
+    ID: 0xfc00,
+    manufacturerCode: 0x153a,
+    attributes: {
+        mode: {ID: 0x0000, type: Zcl.DataType.UINT8, write: true, max: 0xff},
+    },
+    commands: {
+        setMode: {
+            ID: 0x0000,
+            parameters: [{name: "mode", type: Zcl.DataType.UINT8, max: 0xff}],
+        },
+    },
+    commandsResponse: {},
+});
+
+const pilotWireConfig = (configureReporting: boolean): ModernExtend => {
```

Comment:
As far as I can see everything is the same except the `manufacturerCode`, I propose to make it parameter such that the code can be re-used